sonar-kotlin-plugin/src/test/java/org/sonarsource/kotlin/checks/TextBlocksInComplexExpressionsCheckTest.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * SonarSource Kotlin
+ * Copyright (C) 2018-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.kotlin.checks
+
+import org.junit.jupiter.api.Test
+import org.sonarsource.kotlin.verifier.KotlinVerifier
+
+internal class TextBlocksInComplexExpressionsCheckTest : CheckTest(TextBlocksInComplexExpressionsCheck())
+
+@Test
+fun test() {
+    val check = TextBlocksInComplexExpressionsCheck()
+    check.linesNumber = 1
+    KotlinVerifier(check) {
+        fileName = "TextBlocksInComplexExpressionsCheckSampleSingleLines.kt"
+    }.verify()
+}
```

Comment:
I think it would be best to put the test within the test class so that it is displayed in context in the test results.
We can also rename the test to be more explicit in the property to verify
```suggestion
internal class TextBlocksInComplexExpressionsCheckTest : CheckTest(TextBlocksInComplexExpressionsCheck()) {
  @Test
  fun `custom linesNumber rule property is effectively enforced`() {
      val check = TextBlocksInComplexExpressionsCheck()
      check.linesNumber = 1
      KotlinVerifier(check) {
          fileName = "TextBlocksInComplexExpressionsCheckSampleSingleLines.kt"
      }.verify()
  }
}
```